README.md

```diff
@@ -208,6 +208,38 @@ query.data; // Message[]
 
 Queries are cached using a combination of `route name + payload`. So, in the example above, the query key looks roughly like `['GET /messages', { filter: 'some-filter' }]`.
 
+### `useAPIInfiniteQuery`
```

Comment:
I think naming this `useInfiniteAPIQuery` might be more consistent -- then it matches `useCombinedAPIQueries`. Thoughts?